packages/web-components/src/components/va-radio/va-radio.css

```diff
@@ -4,26 +4,21 @@
   display: block;
   border: none;
   padding: 0;
-  margin-top: 3rem;
 }
 
 :host([error]) {
   border-left: 4px solid var(--color-secondary-dark);
-  margin-top: 3rem;
-  padding-bottom: 0.8rem;
-  padding-left: 1.5rem;
-  padding-top: 0.8rem;
+  padding-left: 1.8rem;
   position: relative;
-  right: 1.9rem;
 }
 
 .error-message {
   color: var(--color-secondary-dark);
-  padding-bottom: 3px;
-  padding-top: 3px;
   display: block;
   font-size: 1.6rem;
+  line-height: 2.4rem;
   font-weight: 700;
+  margin-bottom: 1em;
```

Comment:
Should we be using em or rem in these margins? (this one and the one below)

Reply:
@k80bowman @bkjohnson honestly I'm not sure, in some places we use `em` and others we use `rem` I mainly did `1em` for the spacing because that is what is on the p tags and I wanted to make sure the vertical spacing stayed consistent

Reply:
Thanks for bringing this up. Up until now I've been using the existing styles from React components for the most part. I think we should decide on one and prefer `rem` because it's consistent and predictable.

Reply:
I agree with @andresriveratoro, we should prefer `rem` for things like padding and margins. I wanted to ask before recommending that, though, and make sure there wasn't a specific reason were using `em` here.

Reply:
Sounds good switching to `rem`